tape/src/main/java/com/squareup/tape/QueueFile.java

```diff
@@ -134,19 +130,7 @@ private static void writeInt(byte[] buffer, int offset, int value) {
     buffer[offset + 3] = (byte) value;
   }
 
-  /**
-   * Stores int values in buffer. The behavior is equivalent to calling {@link
-   * java.io.RandomAccessFile#writeInt} for each value.
-   */
-  private static void writeInts(byte[] buffer, int... values) {
```

Comment:
This method was wasteful. Caused `int[]` allocations and was always called with the same number of params.